src/workspaces/graphql/mutations.graphql

```diff
@@ -189,3 +189,13 @@ mutation resendWorkspaceInvitation($input: ResendWorkspaceInvitationInput!) {
     errors
   }
 }
+
+mutation generateWebhookPipelineWebhookUrl($input: GeneratePipelineWebhookUrlInput!) {
+  generatePipelineWebhookUrl(input: $input) {
+    success
+    errors
+    pipeline {
```

Comment:
If you add the identifier of the pipeline it's going to automatically save it in the local store